definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js

```diff
@@ -0,0 +1,71 @@
+import { WriteStream } from "fs";
+import { Console } from "console";
+import { EventEmitter } from "events";
+
+declare module "lambda-log" {
+  declare interface LogRecordOptions {
+    level: string;
+    msg: string;
+    meta?: any;
+    tags?: string[];
+  }
+  declare interface LogRecord {
+    msg: string;
+    meta: any;
+    _logLevel: string;
+    _tags: string[];
+  }
+  declare export class LogMessage {
+    level: string;
+    msg: string;
+    meta?: any;
+    tags?: string[];
+    constructor(logRecordOptions: LogRecordOptions, opts: LambdaLogOptions): this;
+    value: LogRecord;
+    log: LogRecord;
+    throw: void;
+    toJSON(format?: boolean): string;
+    static isError(val: any): boolean;
+  }
+  declare interface LambdaLogOptions {
+    meta?: any;
+    tags?: string[];
+    dynamicMeta?: (message: LogMessage) => any;
+    debug?: boolean;
+    dev?: boolean;
+    silent?: boolean;
+    replacer?: (key: string, value: any) => any;
+    stdoutStream?: WriteStream;
+    stderrStream?: WriteStream;
+  }
+  declare interface LogLevels {
+    info: "info";
+    warn: "warn";
+    error: "error";
+    debug: "log" | false;
+    [key: string]: any;
+  }
+  declare export class LambdaLog mixins EventEmitter {
+    LambdaLog: LambdaLog;
+    options: LambdaLogOptions;
+    console: Console;
+  [key: string]: any;
```

Comment:
Same question here.

Reply:
The library allows custom log levels... I changed to statically typed custom log levels in fabdcfa.